pkg/framework/framework.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package framework
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+	errors "golang.org/x/xerrors"
+)
+
+// AriesFramework provides access to clients being managed by the Framework.
+type AriesFramework struct {
```

Comment:
Might be a good idea to name this `Aries` instead

Reply:
@troyronda any thoughts on the name?

Reply:
[Moved](https://github.com/hyperledger/aries-framework-go/compare/9b194c723afdf936279c58a665113ec88f3239e0..94efeb32066ee1f587c67c8642c5e2ff588c5efa#diff-da8e5280b0328cb616f0b42390a35b00R7) the package inside aries folder. The framework can be instantiated with aries.New(opts).